man/systemd-bsod.xml

```diff
@@ -44,6 +44,10 @@
       <xi:include href="standard-options.xml" xpointer="help" />
       <xi:include href="standard-options.xml" xpointer="version" />
     </variablelist>
+
+    <para>The <option> --wait </option> can be used to make systemd-bsod wait indefinitely for a change in the
```

Comment:
use a <varlistentry> - check how other manpages are listing options, for example man/systemd-cat.xml

Reply:
Certainly sir.